test/fulfillmentSpec.js

```diff
@@ -342,9 +409,11 @@ describe('PUT /fulfillment', function () {
     yield this.request()
       .put(transfer.id + '/fulfillment')
       .send(this.cancellationConditionFulfillment)
-      .expect(200)
-      .expect(this.cancellationConditionFulfillment)
-      .expect(validator.validateFulfillment)
+      .expect(422)
+      .expect({
+        id: 'AlreadyRolledBackError',
+        message: 'This transfer has already been rejected'
+      })
```

Comment:
I think canceling a transfer with a `PUT` should be idempotent. It shouldn't return an error, just acknowledge the request as if it were the first one.